### PR TITLE
Update READMEs

### PR DESCRIPTION
**Pull request by clonedagain**
Conforms to the latest README template: bugtracker, runbot etc.
Fixes bugtracker URL on some modules.
States OCA as maintainer, removes other contributors from the 'Maintainer' section.

### CODE REVIEW

**Comment by clonedagain** [comment]
@elicoidal maybe you'll want to check those?

**Comment by pedrobaeza** [comment]
Thanks for taking the time to update this.

:+1:

File: stock_available/README.rst [hunk 1]
```diff
@@ -36,12 +58,14 @@ Contributors
...
+   :target: https://odoo-community.org
 
 This module is maintained by the OCA.
 
-OCA, or the Odoo Community Association, is a nonprofit organization whose mission is to support the collaborative development of Odoo features and promote its widespread use.
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
 
 To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline stock_available/README.rst]: https

File: stock_available_immediately/README.rst [hunk 2]
```diff
@@ -19,12 +43,14 @@ Contributors
...
+   :target: https://odoo-community.org
 
 This module is maintained by the OCA.
 
-OCA, or the Odoo Community Association, is a nonprofit organization whose mission is to support the collaborative development of Odoo features and promote its widespread use.
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
 
 To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline stock_available_immediately/README.rst]: https

File: stock_inventory_preparation_filter/README.rst [hunk 3]
```diff
@@ -10,6 +15,25 @@ Includes more options for making an inventory out of:
 It also allows to make an inventory based on scanned products, adding a line
 with product code and quantity
```
Comment (elicoidal) [inline stock_inventory_preparation_filter/README.rst]: period

File: stock_location_area_data/README.rst [hunk 4]
```diff
@@ -1,47 +1,53 @@
-Features
```
Comment (elicoidal) [inline stock_location_area_data/README.rst]: name of the module in plain English

File: stock_location_area_data/README.rst [hunk 5]
```diff
@@ -1,47 +1,53 @@
...
-Installation
-============
-
-To install this module, you just need to select the module and insure yourself dependencies are available
-
-Configuration
-=============
-
-No particular configuration to use this module
+This module was written to add surface units of measure.
```
Comment (elicoidal) [inline stock_location_area_data/README.rst]: This module adds surface units of measure.

Comment (elicoidal) [inline stock_location_area_data/README.rst]: Besides this should be moved up in the description section of the module

File: stock_location_area_data/README.rst [hunk 6]
```diff
@@ -1,47 +1,53 @@
...
-Maintainer
-==========
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
 
-This module is maintained by Savoir-Faire Linux.
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline stock_location_area_data/README.rst]: https (and any other ;))

File: stock_location_area_management/README.rst [hunk 7]
```diff
@@ -1,50 +1,56 @@
...
-Installation
-============
-
-To install this module, you just need to select the module and insure yourself dependencies are available
-
-Configuration
-=============
-
-No particular configuration to use this module
+This module was written to enter a location's area based on different units of measure.
```
Comment (elicoidal) [inline stock_location_area_management/README.rst]: This module allows to input a location's area based on different units of measure.

File: stock_location_area_management/README.rst [hunk 8]
```diff
@@ -1,50 +1,56 @@
...
+The information depends on the different locations entered.
```
Comment (elicoidal) [inline stock_location_area_management/README.rst]: Which information? vague
s/entered/input (or data)

File: stock_location_ownership/README.rst [hunk 9]
```diff
@@ -1,20 +1,31 @@
...
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
+========================
 Stock Location Ownership
 ========================
 
-Adds an ownership on the stock locations
+Adds an ownership on the stock locations.
```
Comment (elicoidal) [inline stock_location_ownership/README.rst]: Adds ownership to stock locations.

File: stock_mts_mto_rule/README.rst [hunk 10]
```diff
@@ -38,11 +40,27 @@ Usage
 You have to choose the mts+mto route on the product form.
 You should not choose both mts+mto route and mto route.
```
Comment (elicoidal) [inline stock_mts_mto_rule/README.rst]: Where? why?
You need to explicit what the module does and why this is not recommended

File: stock_mts_mto_rule/README.rst [hunk 11]
```diff
@@ -38,11 +40,27 @@ Usage
...
 
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/153/8.0
+
 Configuration
 =============
 
 You have to choose 'Use MTO+MTS rules' on the company's warehouse form.
```
Comment (elicoidal) [inline stock_mts_mto_rule/README.rst]: s/choose/select

File: stock_mts_mto_rule/README.rst [hunk 12]
```diff
@@ -38,11 +40,27 @@ Usage
 You have to choose the mts+mto route on the product form.
```
Comment (elicoidal) [inline stock_mts_mto_rule/README.rst]: s/choose/select

File: stock_quant_manual_assign/README.rst [hunk 13]
```diff
@@ -6,7 +6,7 @@
 Manual assignment of quants
 ===========================
 
-With this module, user can change manually the automatic selection of quants
+With this module, user can change manually the automatic selection of quants.
```
Comment (elicoidal) [inline stock_quant_manual_assign/README.rst]: manually change

File: stock_reserve_sale/README.rst [hunk 14]
```diff
@@ -28,15 +30,24 @@ will get a message suggesting to reserve each line individually. There is no
 module dependency: this modules is fully functional even without ownership
```
Comment (elicoidal) [inline stock_reserve_sale/README.rst:30]: Is this a good idea?
What if you need the module ownership for legacy reason reason (or any other module depending on it)?

**Comment by elicoidal** [comment]
some minor modification except for the ownership remark to be confirmed

File: stock_available/README.rst (hunk 1 above)
Comment (clonedagain) [inline stock_available/README.rst]: OK the fix in template is here can you review it too please? https://github.com/OCA/maintainer-tools/pull/163

**Comment by clonedagain** [comment]
In progress: I'll right the wrongs ASAP and squash it in one commit.

File: stock_location_area_management/README.rst (hunk 8 above)
Comment (clonedagain) [inline stock_location_area_management/README.rst]: No idea what this is about, sorry.
Maybe file a bug report?

File: stock_reserve_sale/README.rst (hunk 14 above)
Comment (clonedagain) [inline stock_reserve_sale/README.rst:30]: No idea, would you file an issue please?

**Comment by clonedagain** [comment]
Changes amended as @elicoidal suggested.

File: stock_location_area_management/README.rst [hunk 15]
```diff
@@ -1,50 +1,56 @@
...
-Credits
-=======
+Maintainer
+----------
 
-* Module developed and tested with Odoo version 8.0
-* For questions, please contact our support services <support@savoirfairelinux.com>
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
```
Comment (clonedagain) [inline stock_location_area_management/README.rst:48]: @savoirfairelinux I hope it's OK for you

File: stock_location_area_data/README.rst [hunk 16]
```diff
@@ -1,47 +1,53 @@
...
-Maintainer
-==========
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
 
-This module is maintained by Savoir-Faire Linux.
+To contribute to this module, please visit https://odoo-community.org.
```
Comment (clonedagain) [inline stock_location_area_data/README.rst:53]: @savoirfairelinux I hope it's OK for you

**Comment by Cedric-Pigeon** [comment]
:+1:  Thanks

**Comment by clonedagain** [comment]
OK to merge?

File: stock_location_area_data/README.rst (hunk 16 above)
Comment (max3903) [inline stock_location_area_data/README.rst:53]: Yes, no problem.

**Comment by max3903** [comment]
:+1:
